release-please-config.json

```
@@ -11,7 +11,7 @@
     "packages/extension-sdk": { },
     "packages/extension-sdk-react": { },
     "packages/extension-utils": { "release-as": "" },
-    "packages/hackathon": { "release-as": ""},
+    "packages/hackathon": { },
     "packages/run-it": { "release-as": "" },
     "packages/sdk": { },
     "packages/sdk-codegen": { "release-as": "" },
```

sdk-codegen got semver bumped to 22.0 but that's not really what we want - gonna pin it back to a minor bump
```suggestion
    "packages/sdk-codegen": { "release-as": "21.4.0" },
```